src/lm/arpa-file-parser.h

```diff
@@ -100,24 +131,28 @@ class ArpaFileParser {
   /// Pure override that must be implemented to process current n-gram. The
   /// n-grams are sent in the file order, which guarantees that all
   /// (k-1)-grams are processed before the first k-gram is.
-  virtual void ConsumeNGram(const NGram&) = 0;
```

Comment:
BTW, I think the Google  style guide forbids consecutive capitals (unless I misunderstand, ???), so it would be
ConsumeNgram.